backend/server/src/campaigns.rs

```diff
@@ -152,7 +166,7 @@ pub async fn roles(
 
     // Prevent people from viewing while it's in draft mode,
     // unless they have adequate permissions
-    if campaign.draft && !user.superuser && permission == AdminLevel::ReadOnly {
+    if campaign.published && !user.superuser && permission == AdminLevel::ReadOnly {
```

Comment:
I think this should be !campaign.published now

Reply:
thanku - good catch, confused as to why the compiler didnt cathc this O.o